### PR TITLE
[ng] src reorg of router example, and convert Jade to .md

### PR DESCRIPTION
**Pull request by chalin**
I've cross checked the original to the markdown-generated version of the page.

Contributes to #713

### CODE REVIEW

File: src/angular/guide/router/3.md [hunk 1]
```diff
@@ -0,0 +1,420 @@
...
+
+While you could continue to add files to the `lib` folder,
+developers generally prefer to organize their apps so that most, if not all,
+files implementing a feature are grouped into a separate folder.
+
+You are about to break up the app into different feature groups, each with its own concerns.
+
+## Add Heroes functionality
+
+Follow these steps:
```
Comment (kwalrath) [inline src/angular/guide/router/3.md]: The following instructions don't take `src` into account.

File: src/angular/guide/router/4.md [hunk 2]
```diff
@@ -0,0 +1,555 @@
...
+<?code-excerpt path-base="router"?>
+{% include_relative _milestone-nav.md selectedOption="4" %}
+
+{% comment %}NOTE: this page currently skips named outlets and the associated functionality of the `ComposeMessageComponent`.{% endcomment %}
+
+It's time to add real functionality to the app's current placeholder crisis center.
+As a quick way to upgrade the basic functionality of the crisis center:
+
+- Delete the placeholder crisis center component file.
+- Create a `lib/crisis-center` folder.
```
Comment (kwalrath) [inline src/angular/guide/router/4.md]: This should be `lib/src/crisis_center`, right?

Aside from the file-copy instructions, the file conversion has looked good so far. I'm going to stop here. Once you've fixed the file-copy instructions throughout this PR, I'll take another look.

File: src/angular/guide/router/3.md (hunk 1 above)
Comment (chalin) [inline src/angular/guide/router/3.md]: Fixed

File: src/angular/guide/router/4.md (hunk 2 above)
Comment (chalin) [inline src/angular/guide/router/4.md]: fixed

File: src/angular/guide/router/3.md [hunk 3]
```diff
@@ -41,16 +41,16 @@ You are about to break up the app into different feature groups, each with its o
 
 Follow these steps:
 
-- Create the `lib/heroes` folder &mdash; you'll be adding files implementing *hero management* there.
-- Copy the following files from [toh-5/lib](https://github.com/angular-examples/toh-5)
-  **into `lib` (_not_ yet `lib/heroes`)**:
+- Create the `lib/src/heroes` folder &mdash; you'll be adding files implementing *hero management* there.
+- Copy the following files from [toh-5/lib/src](https://github.com/angular-examples/toh-5)
```
Comment (kwalrath) [inline src/angular/guide/router/3.md]: toh-5) -> toh-5/src)
?

File: src/angular/guide/router/4.md [hunk 4]
```diff
@@ -19,8 +19,8 @@ It's time to add real functionality to the app's current placeholder crisis cent
 As a quick way to upgrade the basic functionality of the crisis center:
 
 - Delete the placeholder crisis center component file.
-- Create a `lib/crisis-center` folder.
-- Copy the files from `lib/heroes` into the new crisis center folder, but
+- Create a `lib/src/crisis-center` folder.
```
Comment (kwalrath) [inline src/angular/guide/router/4.md]: crisis-center -> crisis_center

File: src/angular/guide/router/3.md (hunk 3 above)
Comment (chalin) [inline src/angular/guide/router/3.md]: The advantage of pointing to the repo, is that the repo landing page has a Download button (which is how folks are most likely to want to get the sources). I'll adjust to be:
> [toh-5](https://github.com/angular-examples/toh-5) `lib/src`

We can always change it later.

File: src/angular/guide/router/4.md (hunk 4 above)
Comment (chalin) [inline src/angular/guide/router/4.md]: Good catch. Fixed.

**Pull request merged**